app/facetest.html

```diff
@@ -0,0 +1,573 @@
+                         <td>Cornell Fencing Team Links:</td>
+                         <td style="padding-right: 5px; padding-left: 5px; width: 40px; height:40px;">
+                              <a href="https://www.facebook.com/CornellFencing/" target="_blank"><img src="./images/facebook.gif"></a>
+                         </td>
+                         <td style="padding-right: 5px; padding-left: 5px; width: 40px; height:40px;">
+                              <a href="https://instagram.com/cornellfencing/" target="_blank"><img src="./images/instagram-icon.gif"></a>
+                         </td>
+                    </tr>
+               </table>
+               </p>
+           </div>
+       </div>
+   </section>
+   <footer>
+
+      <div class="row">
+
+         <div class="twelve columns">
+
+            <!--<ul class="social-links">
+               <li><a href="#"><i class="fa fa-facebook"></i></a></li>
+               <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+               <li><a href="#"><i class="fa fa-instagram"></i></a></li>
+            </ul>-->
+
+             <a class="button" id="member-button" href="https://docs.google.com/forms/d/1VDoX6U44YPvR9snDnxsYTrbYKzNgDiyJuRbIqv82K14/viewform" target="_blank">Hear about CFA events</a>
+
+             <ul class="copyright">
+                 <li>© Copyright 2014 CeeVee</li>
+                 <li>Design by <a title="Styleshout" href="http://www.styleshout.com/" target="_blank">Styleshout</a></li>
+             </ul>
+
+         </div>
+
+         <div id="go-top" style="display: block;"><a class="smoothscroll" style="margin-left:-15px;" title="Back to Top" href="#home"><i class="icon-up-open"></i></a></div>
+
+      </div>
+
+   </footer>
+
+       <!-- Footer End-->
+
+   <!-- Java Script
+   ================================================== -->
+   <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
+   <script>window.jQuery || document.write('<script src="js/jquery-1.10.2.min.js"><\/script>')</script>
+   <script type="text/javascript" src="js/jquery-migrate-1.2.1.min.js"></script>
+
+   <script src="js/jquery.flexslider.js"></script>
+   <script src="js/waypoints.js"></script>
+   <script src="js/jquery.fittext.js"></script>
+   <script src="js/magnific-popup.js"></script>
+
+   <script src="js/modernizr.js"></script>
+   <script src="js/foundation.min.js"></script>
+   <script src="js/foundation.orbit.js"></script>
+   <script src="js/foundation.reveal.js"></script>
+
+   <script>
+       $(document).foundation();
+   </script>
+
+   <script src="js/init.js"></script>
+
+<!-- code from http://www.codingforums.com/dom-and-json-scripting/244531-can-i-do-javascript-not-php.html#post1162842 -->
+<!-- inspiration from https://ranacse05.wordpress.com/2011/02/04/show-facebook-group-wall-on-web-site/ -->
+<script>
+    //lib functions:    (js doesn't come with as much as php...)
+    function aGet(turl,callback){

```

Comment:
So, there is a convenience function for this now, which is called `fetch`. 

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API